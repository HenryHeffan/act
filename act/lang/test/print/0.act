bool x[5];

bool x[7..8];
