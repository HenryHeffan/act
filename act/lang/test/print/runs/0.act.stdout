

/* instances */
bool x[5];
bool x[7..8];

/* connections */
